news/2019-06-15-new-featured-artist-riran.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: RiraN"
+date: 2019-06-15 06:00:00 +0000
+---
+
+osu!'s latest Featured Artist enters the scene in style - hardstyle infact! Meet [**RiraN**](https://osu.ppy.sh/beatmaps/artists/40), one of Asia's up and coming hardstyle artists!
+
+![](https://assets.ppy.sh/artists/40/header.jpg)
+
+We're dedicated towards ensuring a variety of musical styles remain represented in our choice of upcoming Featured Artists, and **RiraN** leads the charge into a sound that many rhythm gamers know and love - hardstyle.
+
+Featured across many other rhythm games, music competitions and compilation albums, **RiraN** has been gradually gaining steam since 2015 in the scene. We're proud to introduce his work to osu! with a selection of **nine** new tracks featuring his signature hardstyle hits, all neatly bundled into less than three minutes per piece.
```

Comment:
Last phrase of first sentence seems odd, would reverse to "gaining steam in the scene since 2015."